content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -74,9 +74,15 @@ If you do not already have a hypervisor installed, install one of these now:
 
 • [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
-{{< note >}}
-Minikube also supports a `--vm-driver=none` option that runs the Kubernetes components on the host and not in a VM. Using this driver requires [Docker](https://www.docker.com/products/docker-desktop) and a Linux environment but not a hypervisor. It is recommended to use the apt installation of docker from [Docker](https://www.docker.com/products/docker-desktop), when using the none driver. The snap installation of docker does not work with minikube.
-{{< /note >}}
+Minikube also supports a `--vm-driver=none` option that runs the Kubernetes components on the host and not in a VM.
+Using this driver requires [Docker](https://www.docker.com/products/docker-desktop) and a Linux environment but not a hypervisor.
+It is recommended to use the apt installation of docker from [Docker](https://www.docker.com/products/docker-desktop), when using the none driver.
+(The snap installation of docker does not work with minikube.)
```

Comment:
I would call out that this advice applies to Debian-style distributions (or maybe just Ubuntu?)
```markdown
If you're using the `none` driver n Debian or a derivatives, you should use the `.deb` packages for
Docker rather than the snap package, which does not work with Minikube.
You can download `.deb` packages from [Docker](https://www.docker.com/products/docker-desktop).
```

Reply:
I'm not sure about the Debian package recommendations. Would it be possible to merge this PR with the caution added, and revisit this?

Reply:
If someone is using an RPM-based system, advising them to install Docker from a `.deb` is likely to confuse them. How's that mitigated?

Reply:
Pushed a commit with your suggested text!